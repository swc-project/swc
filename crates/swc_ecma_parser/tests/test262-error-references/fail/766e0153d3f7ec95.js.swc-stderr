  x Unterminated block comment
   ,-[$DIR/tests/test262-parser/fail/766e0153d3f7ec95.js:1:2]
 1 | `->  
   `----
