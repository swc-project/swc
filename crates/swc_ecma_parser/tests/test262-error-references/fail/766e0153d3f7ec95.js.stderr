
  x Unterminated block comment
   ,-[$DIR/tests/test262-parser/fail/766e0153d3f7ec95.js:2:1]
 2 | /*
   : ^^
   `----
